R/class-accuracy.R

```diff
@@ -82,6 +82,13 @@ accuracy.matrix <- function(data, ...) {
 #' @export
 #' @rdname accuracy
 accuracy_vec <- function(truth, estimate, na_rm = TRUE, case_weights = NULL, ...) {
+  if (is_class_pred(truth)) {
```

Comment:
You don't need to check the observed data, just the predicted classes.

Reply:
Sounds good, I'll change that.

Just for documentation sake, we used to "support" both `truth` and `estimate` as `class_pred` objects
https://github.com/tidymodels/yardstick/blob/d4a9d7af3f44f8dbc8a78827e1ed79b9e46c895d/R/metrics.R#L85-L90

Reply:
It is inappropriate for `truth` to be a `class_pred` object, so we should error in that case.

Reply:
Done in [tidymodels/yardstick@`0352475` (#422)](https://github.com/tidymodels/yardstick/pull/422/commits/0352475d7273b4017465db775728043ceca492d0)